src/components/game.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { Button, Container, Content, Level, Section, SubTitle, Title } from 'reactbulma';
+import GameOptions from '../game/game-options';
+import Board from "./board";
+import GameState from '../game/game-state';
+
+export default class Game extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inProgress: false,
+      currentPlayer: null,
+      playerOne: null,
+      playerTwo: null,
+      tiles: [],
+      gameState: null,
+      lastPosition: '',
+      isOver: false,
+      isWon: false,
+      winner: '',
+    };
```

Comment:
There are a lot of null values in your game state. This is not bad per se, but it does lead me to believe that you might have the wrong abstraction here. Consider how you might refactor this code to remove some of those null values.